x/autopilot/types/parser.go

```diff
@@ -25,13 +25,21 @@ type ModuleRoutingInfo interface {
 	Validate() error
 }
 
+const LiquidStake = "LiquidStake"
+const RedeemStake = "RedeemStake"
+
 // Packet metadata info specific to Stakeibc (e.g. 1-click liquid staking)
 type StakeibcPacketMetadata struct {
-	Action        string `json:"action"`
-	StrideAddress string
+	// TODO: use a constant here
```

Comment:
is this referring to the constants you added above or something else?

Reply:
yes, removed